### PR TITLE
documentation update

### PR DESCRIPTION
**Pull request by Gudnuel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,39 +22,48 @@ This is the offical documentation for a Yarny Social Media platform project buil
...
 
 ```
 
 Navigate to the project directory and install the dependencies:
 
 ```
 
-cd micro-blog
-yarn install next@latest webpack@latest
+cd yarn
```
Comment (todak2000) [inline README.md]: cd yarny not yarn??

**Review by todak2000 — CHANGES_REQUESTED** [review]
please just fix the small changes i noticed. Thanks

File: README.md (hunk 1 above)
Comment (Gudnuel) [reply README.md]: oh.. sorry. yarny is the correct dir name...

Comment (Gudnuel) [reply README.md]: fixed

**Review by Gudnuel — COMMENTED** [review]
fixed

**Review by todak2000 — APPROVED** [review]

**Review by todak2000 — APPROVED** [review]
good to go. thanks

**Pull request merged**